src/rank_llm/rerank/vllm_handler.py

```diff
@@ -23,6 +23,8 @@ def __init__(
             max_logprobs=max_logprobs,
             tensor_parallel_size=tensor_parallel_size,
             gpu_memory_utilization=gpu_memory_utilization,
+            trust_remote_code=True,
```

Comment:
I assume this is needed because of the model architecture?
if yes, please don't hardcode this to true here.
For the sake of consistency, hard code it in https://github.com/castorini/rank_llm/blob/e84b530d7c81cc6c5bc42cb8ca66932ac8c1a276/src/rank_llm/rerank/listwise/rank_listwise_os_llm.py#L157

and if you have time, in a separate PR, please plumb all of them through properly via a new param passed to the RankListwiseOSLLM called vllm_kwargs.